pkg/audit/manager.go

```diff
@@ -359,9 +379,16 @@ func (am *Manager) auditResources(
 					am.log.Error(err, "Unable to list objects for gvk", "group", gv.Group, "version", gv.Version, "kind", kind)
 					continue kindsLoop
 				}
-
+				// for each batch, create a parent folder
+				// prefix kind to avoid delays in removeall
+				subPath = fmt.Sprintf("%s_%d", kind, folderCount)
+				parentDir := path.Join(*apiCacheDir, subPath)
+				if err := os.Mkdir(parentDir, 0o750); err != nil {
```

Comment:
should this match 600 permissions of the files?

Reply:
At a minimum it would need to have the execute bit set for the owner (0700).